### PR TITLE
feat: first troubleshoot

### PR DESCRIPTION
**Pull request by thomasBousselin**

### CODE REVIEW

File: docs/TROUBLESHOOT.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# TROUBLESHOOT
+
+
+## Common Stellio API mistakes
```
Comment (bobeal) [inline docs/TROUBLESHOOT.md:4]:
```suggestion
## Common NGSI-LD API mistakes 
```

File: docs/TROUBLESHOOT.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# TROUBLESHOOT
+
+
+## Common Stellio API mistakes 
+
+### Sending a single entity to a entityOperations endpoint
+
```
Comment (bobeal) [inline docs/TROUBLESHOOT.md:7]: Add an intro sentence like : "when calling an entityOperations endpoints, if you send ..."

File: docs/TROUBLESHOOT.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+```json
+{
+  "type": "https://uri.etsi.org/ngsi-ld/errors/InvalidRequest",
+  "status": 400,
+  "title": "Cannot deserialize value of type `java.util.ArrayList<java.util.Map<java.lang.Object,java.lang.Object>>` from Object value (token `JsonToken.START_OBJECT`)\n at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 1, column: 1]",
+  "instance": "/ngsi-ld/v1/entityOperations/**"
+}
+```
+The endpoint behind `/ngsi-ld/v1/entityOperations` only accepts a list of entities.
```
Comment (bobeal) [inline docs/TROUBLESHOOT.md:16]:
```suggestion
The endpoints behind `/ngsi-ld/v1/entityOperations` only accept a list of entities. 
```

**Review by bobeal — APPROVED** [review]

File: docs/TROUBLESHOOT.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# TROUBLESHOOT
+
+
+## Common NGSI-LD API mistakes 
+
+### Sending a single entity to a entityOperations endpoint
```
Comment (ranim-n) [inline docs/TROUBLESHOOT.md:6]:
```suggestion
### Sending a single entity to an entityOperations endpoint
```

File: docs/TROUBLESHOOT.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    "id": "my:id",
+    "type": "Example",
+    "property1": ...
+  }
+]
+```
+### Other common mistakes
+
+We are looking to document other common mistakes.
+If you're aware of any, feel free to let us know or contribute at https://github.com/stellio-hub/stellio-docs
```
Comment (ranim-n) [inline docs/TROUBLESHOOT.md:33]: maybe say let us know how..opening an issue for example?

File: docs/TROUBLESHOOT.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# TROUBLESHOOT
+
+
+## Common NGSI-LD API mistakes 
+
+### Sending a single entity to a entityOperations endpoint
+
+When calling an entityOperations endpoints, if you send a payload containing an entity you will have the following error :
```
Comment (ranim-n) [inline docs/TROUBLESHOOT.md:8]:
```suggestion
When calling an entityOperations endpoints, if you send a payload containing a single entity you will have the following error : 
```

File: docs/TROUBLESHOOT.md (hunk 5 above)
Comment (thomasBousselin) [inline docs/TROUBLESHOOT.md:33]:
```suggestion
We are looking to document other common mistakes.
If you're aware of any, feel free to let us know on https://github.com/stellio-hub/stellio-docs
```

**Review by ranim-n — APPROVED** [review]

File: docs/TROUBLESHOOT.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
+# TROUBLESHOOT
+
+
+## Common NGSI-LD API mistakes 
+
+### Sending a single entity to an entityOperations endpoint
+
+When calling an entityOperations endpoints, if you send a payload containing a single entity you will have the following error :
```
Comment (bobeal) [inline docs/TROUBLESHOOT.md:8]:
```suggestion
When calling an entityOperations endpoint, if you send a payload containing a single entity you will have the following error: 
```

**Pull request merged**